common/src/main/java/com/microsoft/identity/common/internal/telemetry/AndroidTelemetryContext.java

```diff
@@ -67,9 +68,14 @@ private void addApplicationInfo(@NonNull final Context context) {
             String packageName = "";
 
             if (applicationInfo != null) {
-                applicationName = applicationInfo.labelRes == 0 && applicationInfo.nonLocalizedLabel != null ?
-                        applicationInfo.nonLocalizedLabel.toString() : context.getString(applicationInfo.labelRes);
                 packageName = applicationInfo.packageName;
+
+                if (applicationInfo.labelRes == 0) {
```

Comment:
Does this refer to the <application>label</application> in AndroidManifest file? If yes, can you give some context on how this label is populated for production apps? I was checking MSAL AndroidManifest file and could not find the label tag like it is added in Broker. Even though it is optional,  is there any harm in adding this tag to MSAL as well? And does it resolve the issue in test apps as well?